posix.mak

```diff
@@ -591,7 +591,8 @@ style_lint_shellcmds:
 	grep -nr '[[:blank:]]$$' $$(find etc std -name '*.d'); test $$? -eq 1
 
 	@echo "Enforce whitespace before opening parenthesis"
-	grep -nrE "\<(for|foreach|foreach_reverse|if|while|switch|catch|version)\(" $$(find etc std -name '*.d') ; test $$? -eq 1
+    # do not do whitespace checks on generated unicode tables
+	grep -nrE "\<(for|foreach|foreach_reverse|if|while|switch|catch|version)\(" $$(find etc std -name '*.d' -not -wholename 'std/internal/unicode_*.d') ; test $$? -eq 1
```

Comment:
It shouldn't be too difficult to make the generator emit compliant code (it's just a space before `(`).

If only the generator itself falls foul of code style, then having it in another directory would remove the need for this.

Reply:
Unfortunately, some of the generated files (such as long lines) are indeed a bit hard to fix.

So I'm opting to just disable the checks, as my focus is on the Unicode side of things, not making style checkers happy ;)

Reply:
For dscanner, disabling checks such as line length and others is completely fine. For these crude tests, I can only see one place in the generator that would trigger this.